--- MODS.js	2024-05-28 23:50:32
+++ SafetyLit MODS.js	2024-05-28 23:50:59
@@ -352,6 +352,29 @@
 	var item;
 	let titleInfo;
 	while (item = Zotero.nextItem()) { // eslint-disable-line no-cond-assign
+		// Emiliano Heyns: SafetyLit DOI retrieval
+		let doi
+		if (!item.DOI && (doi = item.url.match(/^(https?:[/][/]([^.]+[.])?doi[.]org[/])?(10.\d{4,9}\/.+)/))) item.DOI = doi[3]
+		// Emiliano Heyns: SafetyLit name fixup
+		for (const creator of (item.creators || [])) {
+			const name = creator.name || (creator.fieldMode === 1 && creator.lastName)
+			if (name && name.replace(/[^a-z ]/i, '').match(/^[a-zA-Z]+ [A-Z]+$/)) {
+				const [given, family] = name.split(' ')
+				creator.fieldMode = 0
+				creator.firstName = given
+				creator.lastName = family.replace(/^(.)(.+)/, (match, car, cdr) => car + cdr.toLowerCase())
+				delete creator.name
+			}
+		}
+		// Emiliano Heyns: SafetyLit cleanup
+		if (item.itemType.endsWith('Article')) {
+			if (`${item.pages}`.match(/^\d+$/)) item.pages = `e${item.pages}`
+			if (!item.pages && !item.volume && !item.issue) item.pages = item.volume = item.issue = 'ePub'
+
+			delete item.journalAbbreviation
+		}
+		// Emiliano Heyns: end of SafetyLit cleanup
+
 		// Don't export notes or standalone attachments
 		if (item.itemType === "note" || item.itemType === "attachment") continue;
 		
